main.js

```diff
@@ -0,0 +1,134 @@
+const listBtn = document.getElementById('call-list');
+const addBooksBtn = document.getElementById('call-add-new');
+const contactBtn = document.getElementById('call-contact');
+const inpTitle = document.getElementById('title');
+const inpAuthor = document.getElementById('author');
+const listSection = document.getElementById('books');
+localStorage.removeItem('randid');
+
+const addBooksSection = document.querySelector('#add-books');
+const contactSection = document.querySelector('footer');
+
+const storage = JSON.parse(localStorage.getItem('BOOkS'));
+// MY OBJECTS //
+let books = [];
+
+function checkStorage() {
+  if (storage !== null) {
+    books = storage;
+  }
+}
+checkStorage();
+class Book {
+  constructor(id, title, author) {
+    this.id = id;
+    this.title = title;
+    this.author = author;
+  }
+
+  Add() {
+    books.push({ id: this.id, title: this.title, author: this.author });
+  }
+
+  Delete() {
+    //  eslint-disable-next-line
+    books = books.filter((item) => {
+      for (let i = 0; i < books.length; i += 1) {
+        if (item.id === this.id) {
+          return false;
+        }
+        return true;
+      }
+    });
+  }
+}
+
+const storeData = () => {
+  localStorage.setItem('BOOkS', JSON.stringify(books));
+};
+
+function lastId() {
+  let lastId = 0;
+  if (books.length > 0) {
+    lastId = books[books.length - 1].id;
+  }
+  return lastId;
+}
+
+let n = lastId();
+
+function printBooks() {
+  let printed = '';
+  //  eslint-disable-next-line
```

Comment:
## OPTIONAL 
* I strongly suggest you fix all linter errors as you can instead of disabling it